src/klipse/klipse_editors.cljs

```diff
@@ -140,15 +140,17 @@
     (when result-element (gdom/setTextContent result-element default-txt))
     (handle-events in-editor
                    {:idle-msec idle-msec
-                    :on-should-eval #(eval-in-html-editor eval-fn result-element in-editor snippet-args state)})
+                    :on-should-eval #((go
+                                        (>! on-eval-channel :reload))
```

Comment:
The default behaviour should be kept as before: evaluate only the edited snippet.
Add a global settings option `reEvaluateAllSnippetsOnChange` (or suggest a better name) to enable this behaviour and document it on the [README settings section](https://github.com/viebel/klipse#configuration).

Reply:
> `reEvaluateAllSnippetsOnChange`

Should I make this option snake case like the others.